packages/http-specs/specs/encode/bytes/main.tsp

```diff
@@ -272,6 +275,9 @@ namespace RequestBody {
     """)
   @post
   op base64(
```

Comment:
@willmtemple I think this one is somewhat unexpected, if I say to encode as base64 we should now treat it as text/plain maybe?

Reply:
It is in fact supposed to do that. The body type is resolved to its lowest encoding target and then _that_ type is used for the mediaTypeHint. Is that not what's happening here?

Reply:
The only effect of the content-type header would be to change it back to JSON, no?

Reply:
oh I guess if it does that above then its all good and this change can be done here ot teh mock api can change to text/plain but this is probably less disruptive.

Reply:
though might be good to add the default case too